docs/quick-start/firmware-options.md

```diff
@@ -45,11 +48,6 @@ Set these to have the "Wifi Update" mode try to connect to an existing wifi netw
 
 ## Other Options
 
-```
-UNLOCK_HIGHER_POWER 
-```
-Enables higher output power for devices that support it but will possibly melt themselves to give it to you. Do not enable it without first updating your cooling setup or verifying the device isn't overheating when running at your chosen power.
-
```

Comment:
Why is this section being removed?

There are tx modules that benefit from this option, like the R9M, ES915TX among others.